Access API/AccessAPI/Controllers/SearchController.cs

```diff
@@ -33,5 +34,45 @@ public class SearchController : ControllerBase
                 return Problem(e.Message);
             }
         }
+
+        [HttpGet] // 127.0.0.1:8081/api/search/categories
+        [Route("/categories")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public ActionResult<List<string>> GetAllCategories([FromQuery] int? limit, [FromQuery] int? offset)
+        {
+            try
+            {
+                return Ok(_sb.CategoriesBll(limit, offset));
+            }
+            catch (SearchQueryResponseException e)
```

Comment:
Det her kunne være en DocumentDataApiException 😎 Den er generel for alle vores controllers, og ikke kun Search, eftersom fejlkoder er ens!

Reply:
Jeg er enig og har ændret navnet på exceptionen.